test/runtests.jl

```diff
@@ -48,8 +48,10 @@ using MixedLayerThermoclineDynamics, OffsetArrays, Test
 end
 
 @time @testset "Field tests" begin
-    nx, ny = 10, 12
-    Lx, Ly = 2.0, 2.4
+
+    include("test_fields.jl")
```

Comment:
I can't run the tests because `test_fields.jl` is not part of this PR.
Is this line left here by accident or does the PR need `test_fields.jl`?

Reply:
I guess it needs it because tests failed when I commented this line out. @dhruvbhagtani, could you push this file?

Reply:
Yes, sorry I forgot to add it. I'll go through the rest of your comments tomorrow, but for now, push the file.

Reply:
Great, I can proceed with my investigations then :)